Remove bare value handling for perspective utilities

adamwathan opened the pull request:
This PR removes bare value handling from the perspective utilities, so that utilities like `perspective-300` will no longer generate declarations like `perspective: 300px`.

Generally we support bare values as a way to avoid putting things into the default theme, and since the perspective utilities already have a named scale in the theme I don't think it makes sense to support bare values. Bare values should be thought of as real API that we would include in the IntelliSense completions list when someone is typing a utility, and I don't think we want to include any numeric values in that list for `perspective` 👍

adamwathan committed:
Remove bare value handling for perspective utilities

RobinMalfait commented on packages/tailwindcss/src/utilities.test.ts:
```diff
@@ -3142,10 +3142,6 @@ test('perspective', () => {
       --perspective-normal: 500px;
     }
 
-    .perspective-123 {
-      perspective: 123px;
-    }
-
```
I think we should move the `perspective-123` example to the expect below where we explicitly test that it doesn't generate anything.

RobinMalfait committed:
move `perspective-123` example

This is now moved to the spot where we ensure that nothing is generated
at all. This prevents us from accidentally updating a snapshot and
missing a potential bug.

RobinMalfait committed:
update changelog
